ci/provisioning/post_provision_config_nodes_UBUNTU_20_04.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+read -r -a inst_rpms <<< "$INST_RPMS"
```

Comment:
(lint) Possible misspelling: INST_RPMS may not be assigned, but inst_rpms is. [SC2153]